_data/guides.yaml

```diff
@@ -250,6 +250,24 @@ categories:
       - title: Amazon Lambda with Vert.x Web, Servlet, or RESTEasy
         url: /guides/amazon-lambda-http
         description: This guide explains how you can deploy Vert.x Web, Servlet, or RESTEasy microservices as an Amazon Lambda.
+      - title: Quarkus Funqy
+        url: /guides/funqy
+        description: This guide explains basics of the Funqy framework
+      - title: Quarkus Funqy HTTP
+        url: /guides/funqy-http
+        description: This guide explains Funqy's HTTP binding
+      - title: Quarkus Funqy Amazon Lambas
+        url: /guides/funqy-amazon-lambda
+        description: This guide explains Funqy's Amazon Lambda binding
+      - title: Quarkus Funqy Amazon Lambas HTTP
+        url: /guides/funqy-amazon-lambda-http
+        description: This guide explains Funqy's Amazon Lambda HTTP binding
+      - title: Quarkus Funqy Knative Events
+        url: /guides/funqy-knative-events
+        description: This guide explains Funqy's Knative Events binding
+      - title: Quarkus Funqy Azure Functions HTTP
+        url: /guides/funqy-azure-functions-http
+        description: This guide explains Funqy's Azure Functions HTTP binding
```

Comment:
Are the other function-related guides still needed? Or should they be dropped?

Talking about `Amazon Lambda with Vert.x Web, Servlet, or RESTEasy` or `Azure Functions (Serverless) with Vert.x Web, Servlet, or RESTEasy` above?

Reply:
They are all still needed.  Funqy is a separate framework.